microreactors/hpmr_h2/cross_sections_library/.gitattributes

```diff
@@ -0,0 +1 @@
+simba_fmh_b_fine.xml filter=lfs diff=lfs merge=lfs -text
```

Comment:
I deleted the checked-in version but I cannot re-add it

@mtnwalker I need push access to your fork of the VTB.
Either that or you pull the latest version of this branch and do `git lfs push --all` after having manually moved (but not git added) the xml file back to the right location

Reply:
@GiudGiud Ok - added you to the fork.

Reply:
ugh still getting
```
batch response: You need Push access to upload Git LFS objects.
```

Reply:
done. thansk!